src/main/java/tests/DataConverterTest.java

```diff
@@ -0,0 +1,61 @@
+package tests;
+
+import core.basesyntax.DataConverter;
+import core.basesyntax.DataConverterImpl;
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterTest {
+
+    private final List<String> rawData = Arrays.asList(
+            "type,fruit,quantity",
+            "b,banana,20",
+            "s,banana,100",
+            "p,banana,13",
+            "r,banana,10"
+    );
+    private final DataConverter dataConverter = new DataConverterImpl();
+    private final List<FruitTransaction> transactions = dataConverter.convertToTransaction(rawData);
```

Comment:
The conversion process is a part of the test, so it should be called within the test methods. This way, each test is responsible for its own data conversion, ensuring that the tests are independent of each other and can be run in any order without affecting the outcome.